Integrations/AttackIQFireDrill/AttackIQFireDrill.py

```diff
@@ -316,6 +317,30 @@ def build_test_results_hr(test_results, test_id):
     return tableToMarkdown(f'Test Results for {test_id}', test_results_mod, keys)
 
 
+def get_page_number_and_page_size(args):
+    """
+    Get arguments page_number and page_size from args
+    Args:
+        args (dict): Argument dictionary, with possible page_number and page_size keys
+
+    Returns (int, int): Return a tuple of (page_number, page_size)
+
+    """
+    page = args.get('page_number', 1)
+    try:
+        page = int(page)
```

Comment:
make sure that page is also natural.(bigger than 1)

Reply:
done and added a check for page_size as well